packages/external-plugin-previewer/src/index.tsx

```diff
@@ -16,7 +16,9 @@ export function MaskExternalPluginPreviewRenderer({ pluginBase, payload, script,
             instance.$mount({ target: dom })
             return () => instance.$destroy()
         } catch (e) {
-            onError?.(e)
+            if (e instanceof Error) {
```

Comment:
This change actually changes the semantics and might break the code. Please always re-throw the error